mods/ra/weapons/other.yaml

```diff
@@ -185,9 +185,12 @@ TTankZap:
 
 DogJaw:
 	ValidTargets: Infantry
-	ReloadDelay: 10
-	Range: 3c0
+	ReloadDelay: 100
+	FireDelay: 200
 	Report: dogg5p.aud
+	Range: 1c512
+	Projectile: Bullet
+		Speed: 1c682
 	Warhead@1Dam: SpreadDamage
```

Comment:
Dogs have small mouths, so spread isn't a great model here.  Someone needs to bite the bullet and add a proper direct-damage warhead (bypassing spread and positions altogether), would you like to volunteer?